website/docs/r/snat.html.markdown

```diff
@@ -11,7 +11,7 @@ description: |-
 Provides a vCloud Director SNAT resource. This can be used to create, modify,
 and delete source NATs to allow vApps to send external traffic.
 
-~> **Note:** This resource may corrupt UI edited NAT rules when used with advanced
+~> **Note:** DEPRECATED: This resource may corrupt UI edited NAT rules when used with advanced
```

Comment:
Not sure is rigth place:) now it seems that deprecated behavior currupting UI:)

Reply:
I think we have piled up the right amount of warnings to users. It's deprecated and we are telling them that it can be dangerous.
We need to point out the danger and we need to inform about deprecation. I believe it is OK as it is